ibtk/src/utilities/HierarchyIntegrator.cpp

```diff
@@ -185,19 +185,19 @@ HierarchyIntegrator::~HierarchyIntegrator()
         d_registered_for_restart = false;
     }
 
-    for (RefinePatchStrategyMap::iterator it = d_ghostfill_strategies.begin(); it != d_ghostfill_strategies.end(); ++it)
+    for (auto& d_ghostfill_strategie : d_ghostfill_strategies)
```

Comment:
This is one of my favorite bugs in clang-modernize.

We should fix these before merging.

Reply:
I thought I fixed these but I must have missed some (maybe a lot?)! I'll see if I can find the rest.